src/core/DynaRec_x64/instructions.cc

```diff
@@ -742,8 +801,18 @@ void DynaRecCPU::recLHU() { recompileLoad<16, false>(); }
 void DynaRecCPU::recLW() { recompileLoad<32, true>(); }
 
 void DynaRecCPU::recLWL() {
-    if (!_Rt_) {
-        error();
+    if (_Rt_ == 0) { // If $rt == 0, just execute the read in case it has side-effects, then return
```

Comment:
The comment here probably should say "most likely", as the compiler will only generate dummy loads for volatiles.